src/gwt/src/org/rstudio/studio/client/application/ui/RStudioThemes.java

```diff
@@ -44,4 +55,35 @@ public static void initializeThemes(String themeName,
    public static boolean isFlat(UIPrefs prefs) {
       return prefs.getFlatTheme().getValue() != "classic"; 
    }
+   
+   private static boolean usesScrollbars() {
+      if (usesScrollbars_ != null) return usesScrollbars_;
+      
+      if (!BrowseCap.isMacintosh()) {
+         usesScrollbars_ = true;
+      }
+      else {
+         Element parent = Document.get().createElement("div");
```

Comment:
To reduce the odds of flicker, we usually set these kinds of test divs to have `position: fixed` and offscreen coordinates.